index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Currency-Converter</title>
+  </head>
+  <body>
+    <div class="texttitle">
+      <h2>Currency Converter</h2>
+    </div>
+
+    <!-- currency-rate-creator part --------------->
+
+    <div class="currency-rate-creator">
+      <form class="currency-rate-creator_form" id="brand-currency-form">
+        <fieldset>
+          <legend>Brand new currency rate ceator</legend>
+
+          <label for="base-currency">The base currency</label>
+          <input
+            type="text"
+            id="base-currency"
+            value="EUR"
+            placeholder="EUR"
+            readonly
+          />
+          <br />
+
+          <div></div>
+
+          <p class="explanations-in-forms">
+            Insert a the quote valuta and rate to be added<br
+              class="zero-rate"
+            />
+          </p>
+
+          <span>
+            <label for="quote-name">Quote name</label>
+            <input
+              type="text"
+              id="quote-name"
+              step="0.001"
+              required
+              placeholder="quote-name"
+            />
+            <div></div>
+
+            <label for="quote-rate">Quote rate</label>
+            <input
+              type="number"
+              id="quote-rate"
+              step="0.001"
+              required
+              placeholder="quote-rate"
+            />
+            <div></div>
+          </span>
+          <br />
+
+          <div class="convert-currency_buttonc-section">
+            <button class="convert-currency_button" type="submit">
+              Add currency & rate
+            </button>
+          </div>
+        </fieldset>
+      </form>
+    </div>
+<BR></BR>
```

Comment:
I think this should be `<br /> <br />` no?

Reply:
Thank you. I have changed that